recipes/flatbuffers/all/conanfile.py

```diff
@@ -43,6 +43,10 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def build_requirements(self):
+        if Version(self.version) >= "2.0.7":
+            self.tool_requires("cmake/3.24.0")
```

Comment:
CMake 3.16 is now the minimum supported version number. They have a legacy CMakeLists.txt for older CMake versions which is unmaintained, however, and does not work.